hummingbird/templates/devices.html

```diff
@@ -0,0 +1,9 @@
+{% if devices %}
+<ul>
+{% for device in devices %}
+<li class="list-group-item">{{ device.mac_id }} <span id="delete_device" data-deviceid="{{ device.id }}" data-userprofile="{{device.user_profile.id}}" class="delete_device glyphicon glyphicon-remove" style="color:red"></span></li>
```

Comment:
This is a long line. Let's split it over several lines.
nit: Indent inside `for` block.